### PR TITLE
feat(input): mobile param

### LINKED ISSUE #1602: Mobile Zooming on Inputs
**Summary**

Currently our input form fields have a text size that is less than 16 px, which causes phones on iOS to zoom into the field when focused/clicked. We should probably set the text size to at least 16 px on mobile to fix this, but there are other possible solutions (see article in additional context section).

**Reproduction**

Steps to reproduce:

1. Navigate to Cashmere on an iPhone
2. Click any input field

Stackblitz reproduction (click the "Edit on Stackblitz" link on a component example and fork the template):

**Additional context**

Here's an article explaining this issue: https://www.warrenchandler.com/2019/04/02/stop-iphones-from-zooming-in-on-form-fields/

### PR DESCRIPTION
**Pull request by andrew-frueh**
flag to increase input font size to prevent zooming on mobile devices

@corykon @aaronbrethorst - added the param we were discussing to the input component.  Also reworked one of the examples on the input page.  We had a redundant one for input suffix, so I made it into an example that demos the different styles available on input.  Let me know if you have any suggestions.

closes #1602

### CODE REVIEW

File: projects/cashmere/src/lib/sass/form-field.scss [hunk 1]
```diff
@@ -145,6 +145,10 @@ $wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
     padding: $infix-padding 10px;
 }
 
+@mixin hc-form-field-infix-mobile() {
+    font-size: calculateRem(16px);
```
Comment (corykon) [inline projects/cashmere/src/lib/sass/form-field.scss:149]: I was a bit confused when I first saw `calculateRem()`. I've been using this mixin everywhere for font size rems: 
`@include fontSize(16px)`. 

Turns out fontSize() mixin just uses calculateRem() under the covers. Not totally sure why we two versions in the library to do the same thing.

So I guess nothing really needs to be changed here... but should figure out why both exist and why we'd use one over the other.

**Review by corykon — APPROVED** [review]

File: projects/cashmere/src/lib/sass/form-field.scss (hunk 1 above)
Comment (andrew-frueh) [reply projects/cashmere/src/lib/sass/form-field.scss:149]: Hah, good point - I've mostly used `fontSize` as well.  In this case I just did a copy/paste from what was being used elsewhere in this stylesheet.  But pretty sure those two functions are totally redundant like you said.

**Comment by aaronbrethorst** [comment]
Sorry I didn't respond sooner. lgtm!

**Pull request merged**